### PR TITLE
Remove optional chaining operator

### LINKED ISSUE #110: Optional chaining in code
The published package contains optional chaining, which is not supported below node v14. 

https://github.com/43081j/eslint-plugin-lit/blob/master/src/rules/attribute-value-entities.ts
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#browser_compatibility

Do you intend to support lower versions of node? Probably good to add an engine field as well to make it clear what's supported.

### PR DESCRIPTION
**Pull request by stramel**
As brought up in the issue, optional chaining operator is not available less than node v14 and node v12 is still in LTS.

Fixes #110

### CODE REVIEW

File: src/rules/attribute-value-entities.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ const rule: Rule.RuleModule = {
                 );
                 const rawValue = analyzer.getRawAttributeValue(element, attr);
 
-                if (!loc || !rawValue?.value) {
+                if (!loc || !(rawValue && rawValue.value)) {
```
Comment (43081j) [inline src/rules/attribute-value-entities.ts:68]: just to be a bit clearer can we go with `if (!loc || !rawValue || !rawValue.value)`

Comment (stramel) [reply src/rules/attribute-value-entities.ts:68]: Yeah thought about that as well. Can do

File: tsconfig.json [hunk 2]
```diff
@@ -1,6 +1,14 @@
 {
   "compilerOptions": {
-    "target": "es2020",
+    // https://github.com/microsoft/TypeScript/wiki/Node-Target-Mapping
+    // https://stackoverflow.com/questions/59787574/typescript-tsconfig-settings-for-node-js-12
+    "lib": [
+      "es2019",
```
Comment (43081j) [inline tsconfig.json:6]: i would just set this as es2020, as the latest 12.x supports all of it afaik

Comment (43081j) [reply tsconfig.json:6]: or at least thats what some quick googling told me 😂  🙊

Comment (stramel) [reply tsconfig.json:6]: Agreed, but that isn't until v12.9+. I figured it would be better to just support all of v12 and use this instead.

**Review by 43081j — APPROVED** [review]

**Pull request merged**